services/ui-src/src/styles/theme.ts

```diff
@@ -3,7 +3,7 @@ import { extendTheme } from "@chakra-ui/react";
 
 export const theme = extendTheme({
   sizes: {
-    appMax: "75rem",
+    appMax: "100vw",
```

Comment:
Since this is what they want and it has to happen somewhere, it might as well happen here.